### DISCUSSION TITLE
Missmatching gid for docker

### DISCUSSION THREAD

**Discussion by volker-raschek** [renovatebot/docker-renovate#738] [Q&A] [answered]
Hi, 
I am trying to start renovate via docker and receive errors that renovate can not communicate with docker. The reason is, that the underlying arch linux distribution use the gid 997 by default , but the ubuntu user of the renovate container is part of the docker group with the gid 999. 

 ```bash
$ getent group docker
docker:x:977:docker_users,markus,root

$ docker run -it --rm --entrypoint sh --volume /var/run/docker.sock:/var/run/docker.sock docker.io/renovate/renovate:31.97.3-slim
$ id
uid=1000(ubuntu) gid=0(root) groups=0(root),999(docker),1000(ubuntu)
``` 

It there a way to configure the gid via startup?

**Comment by viceice** [comment, marked as answer]
Run with `-u 1000:997`. Checkout the docker `--user` command.

**Comment by bug-tape** [comment]
Same problem with new ubuntu, group id changed to 998.
Building own renovate image for now to add the group with this `Dockerfile`:
```Dockerfile
FROM renovate/renovate:<your version>

# Switch to root user
USER 0
# Create extra docker group matching needed gid.
RUN addgroup --gid 998 docker-ext
# Add extra group to user used by renovate.
RUN adduser ubuntu docker-ext
# Switch back to renovate user.
USER 1000
```

**Comment by viceice** [comment]
Did you tried my suggestion?

**Comment by volker-raschek** [comment]
Hi @viceice, 
yes and it works fine. Thanks a lot (:
